pkg/testing/tools/cmd.go

```diff
@@ -18,8 +18,8 @@ func InstallAgent(installOpts atesting.InstallOpts, agentFixture *atesting.Fixtu
 // InstallStandaloneAgent force install the Elastic Agent through agentFixture.
 func InstallStandaloneAgent(agentFixture *atesting.Fixture) ([]byte, error) {
 	installOpts := atesting.InstallOpts{
-		NonInteractive: true,
-		Force:          true,
+		// NonInteractive: true,
```

Comment:
Uncomment or remove the line entirely?

Reply:
I can undo the change now, since in some of the rebases I switched the install to the new signature a few lines above. Good catch!